_data/footer.yaml

```diff
@@ -0,0 +1,61 @@
+socialLinks:
+  - icon: linkedin
+    href: https://www.linkedin.com/company/red-hat
+    content: LinkedIn
+  - icon: youtube
+    href: https://www.youtube.com/user/RedHatVideos
+    content: Youtube
+  - icon: facebook
+    href: https://www.facebook.com/redhatinc
+    content: Facebook
+  - icon: twitter
+    href: https://twitter.com/RedHat
+    content: Twitter
+  - icon: github
+    href: https://github.com/redhat-israel
+    content: GitHub
+
+links:
+  - heading: המוצרים שלנו
+    links:
+    - href: https://www.redhat.com/en/technologies/management/ansible
+      content: Red Hat Ansible Automation Platform
+    - href: https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux
+      content: Red Hat Enterprise Linux
+    - href: https://www.redhat.com/en/technologies/cloud-computing/openshift
+      content: Red Hat OpenShift
+    - href: https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation
+      content: Red Hat OpenShift Data Foundation
+    - href: https://www.redhat.com/en/technologies/linux-platforms/openstack-platform
+      content: Red Hat OpenStack Platform
+    - href: https://www.redhat.com/en/technologies/all-products
+      content: See all products
```

Comment:
Do we need this for beyond?

Reply:
we can either:
1. replace the footer content/links with something more appropriate 
2. remove the main footer and use only the global (i.e. legal) footer

Reply:
can we leave this for a subsequent PR?